libafl/src/fuzzer/mod.rs

```diff
@@ -411,7 +411,7 @@ where
     where
         EM: EventFirer<State = Self::State>,
     {
-        let exec_res = self.execute_no_process(state, manager, &input, observers, exit_kind)?;
+        let exec_res = self.check_results(state, manager, &input, observers, exit_kind)?;
         let corpus_id = self.process_execution(
```

Comment:
Shouldn't this guy be `evaluate_execution`?

Reply:
evaluate_execution is already used. actually it's wrapping this

evaluate_execution have two wrapped functions 
check_results: to see if the testcase is obj or corpus and
process_execution: append metadatas

Reply:
naming is hard